### PR TITLE
Fixes weird inconsistencies and makes use of switch expressions instead of if chains

### PR DESCRIPTION
**Pull request by nobbele**
lol

### CODE REVIEW

File: osu.Game.Rulesets.Diva/Difficulty/DivaDifficultyCalculator.cs [hunk 1]
```diff
@@ -18,27 +18,24 @@ public DivaDifficultyCalculator(Ruleset ruleset, WorkingBeatmap beatmap)
         {
         }
 
-        protected override DifficultyAttributes CreateDifficultyAttributes(IBeatmap beatmap, Mod[] mods, Skill[] skills, double clockRate) {
+        protected override DifficultyAttributes CreateDifficultyAttributes(IBeatmap beatmap, Mod[] mods, Skill[] skills, double clockRate)
+        {
```
Comment (Thesola10) [inline osu.Game.Rulesets.Diva/Difficulty/DivaDifficultyCalculator.cs:22]: Moving the brace on its own line is personal preference, check with @Artemis-chan if they're okay with that

File: osu.Game.Rulesets.Diva/DivaRuleset.cs [hunk 2]
```diff
@@ -139,7 +139,7 @@ public override IEnumerable<Mod> GetModsFor(ModType type)
             new KeyBinding(InputKey.W, DivaAction.Triangle),
             new KeyBinding(InputKey.S, DivaAction.Cross),
             new KeyBinding(InputKey.D, DivaAction.Circle),
-            
+
```
Comment (Thesola10) [inline osu.Game.Rulesets.Diva/DivaRuleset.cs:142]: I struggle to see the change on mobile, what is that empty line?

File: osu.Game.Rulesets.Diva/Scoring/DivaHitWindows.cs [hunk 3]
```diff
@@ -5,16 +5,17 @@ namespace osu.Game.Rulesets.Diva.Scoring
...
-            case HitResult.Meh:
-            case HitResult.Miss:
-                return true;
+                case HitResult.Perfect:
+                case HitResult.Great:
+                case HitResult.Good:
+                case HitResult.Ok:
+                case HitResult.Meh:
+                case HitResult.Miss:
+                    return true;
```
Comment (Thesola10) [inline osu.Game.Rulesets.Diva/Scoring/DivaHitWindows.cs:18]: Again, personal preference when it comes to switch block indent, I personally disagree with this particular style

**Review by Thesola10 — COMMENTED** [review]
I don't see much in terms of functional changes, this PR seems mostly stylistic. I haven't had the chance to review the code style fully myself, so this is appreciated.

File: osu.Game.Rulesets.Diva/DivaRuleset.cs (hunk 2 above)
Comment (nobbele) [reply osu.Game.Rulesets.Diva/DivaRuleset.cs:142]: line with spaces to empty line

File: osu.Game.Rulesets.Diva/Difficulty/DivaDifficultyCalculator.cs (hunk 1 above)
Comment (nobbele) [reply osu.Game.Rulesets.Diva/Difficulty/DivaDifficultyCalculator.cs:22]: it's on a new line everywhere else

Comment (Thesola10) [reply osu.Game.Rulesets.Diva/Difficulty/DivaDifficultyCalculator.cs:22]: Ah, figures.

**Comment by nobbele** [comment]
> 
> 
> I don't see much in terms of functional changes, this PR seems mostly stylistic. I haven't had the chance to review the code style fully myself, so this is appreciated.

Might try to do some functional changes later

**Comment by Artemis-chan** [comment]
yea it's all good

**Pull request merged**